curator-x-discovery/src/main/java/org/apache/curator/x/discovery/ServiceProviderBuilder.java

```diff
@@ -71,4 +74,23 @@
      * @return this
      */
     public ServiceProviderBuilder<T> additionalFilter(InstanceFilter<T> filter);
+
+    /**
+     * Optional ExecutorService to use for the cache's background thread. The specified ExecutorService
+     * will be wrapped in a CloseableExecutorService and overrides any prior ThreadFactory or ExecutorService
```

Comment:
Should be `... and overrides any prior ThreadFactory or CloseableExecutorService` (instead of `ExecutorService`), no?
Same question for next method.

Reply:
done for both.